verify/main.go

```diff
@@ -135,14 +135,14 @@ func verifyCorrection(correction []string, db, ispdb *geoip2.Reader, asnCounts m
 		return "", fmt.Errorf("unable to find city record for %s: %w", networkOrIP, err)
 	}
 
-	firstSubdivision := ""
+	lastSubdivision := ""
```

Comment:
Do you mind changing the name to something like `mostSpecificSubdivision` to make it more clear why we have chosen this one?